dev-support/ranger-docker/README.md

```diff
@@ -40,6 +40,7 @@ Docker files in this folder create docker images and run them to build Apache Ra
    export DOCKER_BUILDKIT=1
    export COMPOSE_DOCKER_CLI_BUILD=1
    export RANGER_DB_TYPE=postgres
+   export TARGETARCH=arm64
```

Comment:
@kumaab  - TARGETARCH will be automatically set if following env is set while building the image:

`DOCKER_BUILDKIT=1 `

More on this at https://docs.docker.com/build/buildkit/ and https://docs.docker.com/engine/reference/builder/.

Reply:
Got it, thank you!